### DISCUSSION TITLE
Stopping all mocks

### DISCUSSION THREAD

**Discussion by SergeyKrivorotko1** [mockoon/mockoon#1424] [General]
How can I stop all mocks or restart them? I previously used the mockoon-cli stop "all" command, but it was removed in version 4.0.0. If I restart the mocks with the start command, I get an error that the ports cannot be reused

**Comment by 255kb** [comment]
The CLI is now started by default as a [foreground process](https://github.com/mockoon/mockoon/blob/main/packages/cli/README.md#start-command). If you run the following command from your terminal `mockoon-cli start ... ` you will have to kill the process with ctrl+c before starting a new one.

**Reply by SergeyKrivorotko1** [reply]
We are running the command from the terminal in the background (add mockoon-cli start ... & to the command) and we need to also stop in the background

**Reply by 255kb** [reply]
The `&` is something from your operating system, not from Mockoon CLI. 
Usually when you use the `&` you will get a process id (like 1234).
You can then kill the process by doing a `kill 1234`.
